### ISSUE TITLE
Installing from source fails with `cannot use h.allow0RTT`

### ISSUE THREAD

**Issue by Crocmagnon** [mr-karan/doggo#79]
```console
$ go install github.com/mr-karan/doggo/cmd/doggo@latest
# github.com/quic-go/quic-go/internal/handshake
../../go/pkg/mod/github.com/quic-go/quic-go@v0.37.0/internal/handshake/crypto_setup.go:334:37: cannot use h.allow0RTT (variable of type bool) as tls.QUICSessionTicketOptions value in argument to h.conn.SendSessionTicket

$ go version
go version go1.21.0 darwin/arm64
```

**Comment by jlucktay** [comment]
Looks like 9818c32 fixes this, and just needs to be tagged/released.

**Comment by mr-karan** [comment]
Released - [v0.5.7](https://github.com/mr-karan/doggo/releases/tag/v0.5.7) :)

**Comment by Crocmagnon** [comment]
I confirm the issue is fixed, thanks 👍🏻
